### PR TITLE
Include symbols for Tasks and Tools projects in Razor.Design

### PR DESCRIPTION
**Pull request by pranavkm**
Fixes https://github.com/aspnet/Razor/issues/2215

### CODE REVIEW

File: src/Microsoft.AspNetCore.Razor.Design/Microsoft.AspNetCore.Razor.Design.csproj [hunk 1]
```diff
@@ -57,24 +57,34 @@
...
       <Output TaskParameter="CopiedFiles" ItemName="FileWrites" />
     </Copy>
-    <Copy SourceFiles="@(TaskAssemblyNetStandard)" DestinationFolder="$(OutputPath)\tasks\netstandard2.0\">
+    <Copy SourceFiles="@(TaskAssemblyNetStandard);@(TaskSymbolNetStandard)" DestinationFolder="$(OutputPath)\tasks\netstandard2.0\">
       <Output TaskParameter="CopiedFiles" ItemName="FileWrites" />
     </Copy>
 
     <ItemGroup>
       <None Include="@(TaskAssemblyNet46)" PackagePath="tasks\net46\" Pack="true" />
+      <None Include="@(TaskSymbolNet46)" PackagePath="tasks\net46\" Pack="true" />
```
Comment (pranavkm) [inline src/Microsoft.AspNetCore.Razor.Design/Microsoft.AspNetCore.Razor.Design.csproj]: @rynowak we couldn't find any obvious way to produce a symbol nupkg, specify a package path to an item while excluding it only in the main assembly. The result is that we have pdbs sitting in the Design package and it's symbol nupkg. Seems like a small price to pay for compliance. We could open a separate issue to address this in rc if you'd like.

Comment (rynowak) [reply src/Microsoft.AspNetCore.Razor.Design/Microsoft.AspNetCore.Razor.Design.csproj]: Sorry what's the implication of this? I'm not in charge of decisions like this.

Comment (ajaybhargavb) [reply src/Microsoft.AspNetCore.Razor.Design/Microsoft.AspNetCore.Razor.Design.csproj]: The implication is after this change we'll have pdbs sitting in both `Microsoft.AspNetCore.Razor.Design.nupkg` and `Microsoft.AspNetCore.Razor.Design.Symbols.nupkg`.

**Comment by pranavkm** [comment]
Spoke to @Eilon - he'd rather not us ship the pdbs in the main nupkg. He's also 🆗  with us not having symbols for preview2 as long as we do the right thing for rc. Closing this, but I'll leave the branch alone. @ajaybhargavb re-assigning the issue to you so you could take a stab at doing the right thing ™️

**Comment by rynowak** [comment]
Should we just 'pack' the tasks package as non-shipping? That sounds a like a great solution to this.

**Comment by pranavkm** [comment]
Not sure if that works for how they do symbol publishing today (which is look for a symbol package per regular nupkg).

**Pull request closed without merging**